tests/cypress/src/DashTable.ts

```diff
@@ -19,104 +19,200 @@ const getSelector = (state: State) => {
     }
 };
 
-export default class DashTable {
-    static getCell(row: number, column: number, editable: State = State.Ready) {
-        return cy.get(`#table ${getSelector(editable)} tbody tr td.column-${column}`).eq(row);
+export class DashTableHelper {
```

Comment:
Generalize the helper to work for multiple tables - Leaving the static implementation below for existing tests